desc/coils.py

```diff
@@ -1492,6 +1529,27 @@ def to_SplineXYZ(self, knots=None, grid=None, method="cubic", name=""):
         coils = [coil.to_SplineXYZ(knots, grid, method) for coil in self]
         return self.__class__(*coils, NFP=self.NFP, sym=self.sym, name=name)
 
+    def compute_minimum_intercoil_distances(self, grid=None):
```

Comment:
we had talked about this before and while I agree something like this is necessary, I don't know if we want to commit to this API at the moment.

For the current purpose, it might be simpler to make it something more like `eq.is_nested` where we just check if there are any likely intersections, returning a bool and raising a warning.

Reply:
I will make the change tmrw, if anyone wants to beat me to it go ahead

I will just keep the same exact fxn but add a threshold argument and have it return a bool if any of the distances are below the threshold (maybe also returning the indices of the offending coils)